packages/ckeditor5-dev-env/lib/release-tools/tasks/releasesubrepositories.js

```diff
@@ -208,28 +220,79 @@ module.exports = function releaseSubRepositories( options ) {
 		} );
 	}
 
-	function releasePackages( releaseOptions ) {
+	function pullAndPushPackages() {
```

Comment:
Why is this function located before `bumpVersion()`? Sort the code in the order your read it.